aldryn_jobs/south_migrations/0033_auto__del_field_jobapplication_job_offer__add_field_jobapplication_job.py

```diff
@@ -0,0 +1,216 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Deleting field 'JobApplication.job_offer'
+        db.delete_column(u'aldryn_jobs_jobapplication', 'job_offer_id')
```

Comment:
why not rename column (http://south.readthedocs.org/en/latest/databaseapi.html#db-rename-column) ?